src/components/publicApi/types.ts

```diff
@@ -1 +1,24 @@
 export type InitialRouteName = 'login' | 'login-password' | 'new-account'
+
+/**
+ * Subset of AppConfig from edge-react-gui
+ */
+export interface AppConfig {
+  // appId?: string
+  // appName: string
+  // appNameShort: string
+  // appStore: string
+  // configName: string
+  // darkTheme: Theme
+  // defaultWallets: string[]
+  // knowledgeBase: string
+  // lightTheme: Theme
+  // notificationServers: string[]
+  // phoneNumber: string
+  // referralServers?: string[]
+  // supportsEdgeLogin: boolean
+  // supportEmail: string
+  // supportSite: string
+  termsOfServiceSite: string
+  // website: string
+}
```

Comment:
This is odd. Why are these being committed out from the type? Can we just remove them instead?

Reply:
They are a reference in case we want to use any of these in the login-ui. I omitted values that we would never use in the login-ui